src/djimatrice_drone.py

```diff
@@ -0,0 +1,230 @@
+import roslibpy
+import numpy as np
+from drone import Drone
+from enum import Enum
+
+class DjiMatriceDrone(Drone):
+
+    drone_type = "DjiMatrice"
```

Comment:
We should change either this drone_type value to "DjiMatriceDrone" or we should change the mavros_drone.py drone_type value to "Mavros" to stay consistent.

Reply:
Lets just say Mavros for sake of brevity. I'll add to the Mavros end.